packages/safe-ds-lang/src/language/runner/safe-ds-runner.ts

```diff
@@ -283,11 +282,13 @@ export class SafeDsRunner {
      *
      * @param pipelineDocument Document containing the main Safe-DS pipeline to execute.
      * @param id A unique id that is used in further communication with this pipeline.
+     * @param pipelineName Name of the pipeline that should be run
      * @param targetPlaceholder The name of the target placeholder, used to do partial execution. If no value or undefined is provided, the entire pipeline is run.
      */
     public async executePipeline(
         pipelineDocument: LangiumDocument,
         id: string,
+        pipelineName: string,
         targetPlaceholder: string | undefined = undefined,
```

Comment:
```suggestion
        id: string,
        pipelineDocument: LangiumDocument,
        pipelineName: string,
        targetPlaceholder: string | undefined = undefined,
```

I'd like to have the parameters for documents, pipeline and placeholder next to each other. Since `targetPlaceholder` is optional, this is the only way.